### PR TITLE
Qt: Update graphics config descriptions for consistency

### PR DESCRIPTION
**Pull request by 8times9**
I've updated the graphics config descriptions for overall consistency. They've been edited for the format:

- Brief sentence describing what the option does (with the verb in the form of what the option itself does as opposed to what the user can do with the option)
- 2 line breaks (one blank line in between)
- Further notes to user (with necessary line breaks between)
- 2 line breaks
- Brief sentence recommending the best setting

I also removed any contractions and made the descriptions more concise where I could. I didn't touch the ubershaders descriptions as IIRC there was already a lengthy discussion over the best wording for them.

Additionally on the Post-Processing config window I added a dash in the title.

### CODE REVIEW

File: Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp [hunk 1]
```diff
@@ -273,74 +273,73 @@ void EnhancementsWidget::SaveSettings()
...
-      QT_TR_NOOP("Disables the blending of adjacent rows when copying the EFB. This is known in "
-                 "some games as \"deflickering\" or \"smoothing\". Disabling the filter has no "
-                 "effect on performance, but may result in a sharper image, and causes few "
-                 "graphical issues.\n\n\nIf unsure, leave this checked.");
+                 "cause issues in others.\n\nIf unsure, leave unchecked.");
+  static const char TR_DISABLE_COPY_FILTER_DESCRIPTION[] = QT_TR_NOOP(
+      "Disables the blending of adjacent rows when copying the EFB.\n\nThis is known "
+      "in some games as \"deflickering\" or \"smoothing\". Disabling the filter has no "
+      "effect on performance, but may result in a sharper image, and causes few "
+      "graphical issues.\n\n\nIf unsure, leave checked.");
```
Comment (Techjar) [inline Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: You missed this triple line break.

**Comment by Techjar** [comment]
The convention across all the descriptions is to use a double line break *only* before the "If unsure..." sentence. The reason for this is (presumably) to prevent the descriptions from being too tall for the description pane at the default window size.

File: Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp [hunk 2]
```diff
@@ -273,74 +273,73 @@ void EnhancementsWidget::SaveSettings()
...
       "Calculates lighting of 3D objects per-pixel rather than per-vertex, smoothing out the "
-      "appearance of lit polygons and making individual triangles less noticeable.\nRarely causes "
-      "slowdowns or graphical issues.\n\nIf unsure, leave this unchecked.");
+      "appearance of lit polygons and making individual triangles less noticeable.\n\nRarely "
+      "causes slowdowns or graphical issues.\n\nIf unsure, leave this unchecked.");
   static const char TR_WIDESCREEN_HACK_DESCRIPTION[] = QT_TR_NOOP(
-      "Forces the game to output graphics for any aspect ratio.\nUse with \"Aspect Ratio\" set to "
-      "\"Force 16:9\" to force 4:3-only games to run at 16:9.\nRarely produces good results and "
-      "often partially breaks graphics and game UIs.\nUnnecessary (and detrimental) if using any "
+      "Forces the game to output graphics at any given aspect ratio.\n\nUse with \"Aspect Ratio\" "
```
[the remark this answers is not in the document]
Comment (8times9) [reply Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: new description
It actually fits fine.

[the remark this answers is not in the document]
Comment (Techjar) [reply Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: Fonts are platform-dependent and it will not appear the same on other operating systems.

[the remark this answers is not in the document]
Comment (Techjar) [reply Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: Example from my Linux install with KDE:
image

[the remark this answers is not in the document]
Comment (8times9) [reply Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: Okay, I removed the line in between the first line and "Auto:" entry. Is that enough room? I can't really make it shorter

[the remark this answers is not in the document]
Comment (Techjar) [reply Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: That isn't the section I'm referring to, that one looks fine. Widescreen hack is much too tall, however.

[the remark this answers is not in the document]
Comment (8times9) [reply Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: Whoops you're right, sorry. 

I pushed a commit, let me know if it's better now.

[the remark this answers is not in the document]
Comment (Techjar) [reply Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: Yes this one looks fine now.

File: Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp [hunk 3]
```diff
@@ -273,74 +273,73 @@ void EnhancementsWidget::SaveSettings()
...
+      "and often partially breaks graphics and game UIs.\nUnnecessary (and detrimental) if using any "
       "AR/Gecko-code widescreen patches.\n\nIf unsure, leave this unchecked.");
   static const char TR_REMOVE_FOG_DESCRIPTION[] =
-      QT_TR_NOOP("Makes distant objects more visible by removing fog, thus increasing the overall "
-                 "detail.\nDisabling fog will break some games which rely on proper fog "
-                 "emulation.\n\nIf unsure, leave this unchecked.");
+      QT_TR_NOOP("Makes distant objects more visible by removing fog, thus increasing overall "
+                 "detail.\n\nDisabling fog will break some games which rely on proper fog "
+                 "emulation.\n\nIf unsure, leave unchecked.");
   static const char TR_3D_MODE_DESCRIPTION[] = QT_TR_NOOP(
```
Comment (Techjar) [inline Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: This description is too many lines as well.
img

Comment (8times9) [reply Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: I removed some line breaks, I can remove one more if you need me to.

File: Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp [hunk 4]
```diff
@@ -273,74 +273,73 @@ void EnhancementsWidget::SaveSettings()
...
-                 "issues. SSAA is significantly more demanding than MSAA, but provides top quality "
-                 "geometry anti-aliasing and also applies anti-aliasing to lighting, shader "
-                 "effects, and textures.\n\nIf unsure, select None.");
+  static const char TR_INTERNAL_RESOLUTION_DESCRIPTION[] = QT_TR_NOOP(
+      "Specifies the resolution used to render at.\n\nA higher resolution greatly improves "
+      "visual quality, but also greatly increases GPU load and can cause issues in "
+      "certain games. Generally speaking, the lower the internal resolution is, the "
+      "better your performance will be.\n\nIf unsure, select Native.");
+
+  static const char TR_ANTIALIAS_DESCRIPTION[] = QT_TR_NOOP(
```
Comment (Techjar) [inline Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: Went over by a few pixels here.
img

File: Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp [hunk 5]
```diff
@@ -142,62 +142,62 @@ void AdvancedWidget::OnEmulationStateChanged(bool running)
...
+      "\n\nIf the aspect ratio is widescreen, the output "
       "image will be scaled horizontally to preserve the vertical resolution.\n\nIf unsure, leave "
-      "this unchecked.");
+      "unchecked.");
 #if defined(HAVE_FFMPEG)
   static const char TR_USE_FFV1_DESCRIPTION[] =
-      QT_TR_NOOP("Encode frame dumps using the FFV1 codec.\n\nIf unsure, leave this unchecked.");
+      QT_TR_NOOP("Encodes frame dumps using the FFV1 codec.\n\nIf unsure, leave unchecked.");
 #endif
   static const char TR_FREE_LOOK_DESCRIPTION[] = QT_TR_NOOP(
```
Comment (Techjar) [inline Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp]: This one too.
img

File: Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp [hunk 6]
```diff
@@ -190,53 +190,54 @@ void HacksWidget::SaveSettings()
...
   static const char TR_ACCUARCY_DESCRIPTION[] = QT_TR_NOOP(
-      "The \"Safe\" setting eliminates the likelihood of the GPU missing texture updates "
-      "from RAM.\nLower accuracies cause in-game text to appear garbled in certain "
-      "games.\n\nIf unsure, use the rightmost value.");
+      "Adjusts the accuracy at which the GPU receives texture updates from RAM. "
+      "\n\nThe \"Safe\" setting eliminates the likelihood of the GPU missing texture updates "
+      "from RAM.\n\nLower accuracies cause in-game text to appear garbled in certain "
+      "games.\n\nIf unsure, select the rightmost value.");
 
   static const char TR_STORE_XFB_TO_TEXTURE_DESCRIPTION[] = QT_TR_NOOP(
```
Comment (Techjar) [inline Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp]: Again, a few pixels.
img

**Comment by Techjar** [comment]
I think the addition of some of the double line breaks does improve readability. Perhaps remove some of the single line breaks in those instances to reduce the line count?

File: Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp [hunk 7]
```diff
@@ -142,62 +142,62 @@ void AdvancedWidget::OnEmulationStateChanged(bool running)
...
+      "speed.\n\nIf unsure, leave unchecked.");
   static const char TR_CROPPING_DESCRIPTION[] =
-      QT_TR_NOOP("Crop the picture from its native aspect ratio to 4:3 or "
-                 "16:9.\n\nIf unsure, leave this unchecked.");
+      QT_TR_NOOP("Crops the picture from its native aspect ratio to 4:3 or "
+                 "16:9.\n\nIf unsure, leave unchecked.");
   static const char TR_PROGRESSIVE_SCAN_DESCRIPTION[] = QT_TR_NOOP(
-      "Enables progressive scan if supported by the emulated software.\nMost games don't "
-      "care about this.\n\nIf unsure, leave this unchecked.");
+      "Enables progressive scan if supported by the emulated software.\n\nMost "
```
Comment (Techjar) [inline Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp]: We could probably just not have a line break here. It's only 2 sentences.

**Comment by MayImilae** [comment]
It's progress report time so I can't really spend the time to look this over. I'll get to reviewing this soon!

File: Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp [hunk 8]
```diff
@@ -142,62 +142,62 @@ void AdvancedWidget::OnEmulationStateChanged(bool running)
...
+      QT_TR_NOOP("Renders the scene as a wireframe.\n\nIf unsure, leave unchecked.");
   static const char TR_SHOW_STATS_DESCRIPTION[] =
-      QT_TR_NOOP("Show various rendering statistics.\n\nIf unsure, leave this unchecked.");
-  static const char TR_TEXTURE_FORMAT_DECRIPTION[] =
-      QT_TR_NOOP("Modify textures to show the format they're encoded in. Needs an emulation reset "
-                 "in most cases.\n\nIf unsure, leave this unchecked.");
+      QT_TR_NOOP("Shows various rendering statistics.\n\nIf unsure, leave unchecked.");
+  static const char TR_TEXTURE_FORMAT_DECRIPTION[] = QT_TR_NOOP(
+      "Modifies textures to show the format that they are encoded in.\n\nNeeds "
+      "an emulation reset in most cases.\n\nIf unsure, leave unchecked.");
```
Comment (Techjar) [inline Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp]: Does this actually need an emulation reset? Can someone verify this? I've toggled the option after game start with success before.

Comment (8times9) [reply Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp]: We could reword it to "May require an emulation reset to apply"

Comment (Techjar) [reply Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp]: I'm happy with that.

**Comment by 8times9** [comment]
@Techjar I pushed another commit that hopefully fixes all of the pixel-length errors.

File: Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp [hunk 9]
```diff
@@ -142,62 +142,62 @@ void AdvancedWidget::OnEmulationStateChanged(bool running)
...
   static const char TR_USE_FFV1_DESCRIPTION[] =
-      QT_TR_NOOP("Encode frame dumps using the FFV1 codec.\n\nIf unsure, leave this unchecked.");
+      QT_TR_NOOP("Encodes frame dumps using the FFV1 codec.\n\nIf unsure, leave unchecked.");
 #endif
   static const char TR_FREE_LOOK_DESCRIPTION[] = QT_TR_NOOP(
-      "This feature allows you to change the game's camera.\nMove the mouse while holding the "
-      "right "
-      "mouse button to pan and while holding the middle button to move.\nHold SHIFT and press "
+      "Allows the user to manipulate the game camera.\n\nMove "
+      "the mouse while holding the right "
```
Comment (Techjar) [inline Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp]: I think this would be much more concise and readable:

Move the mouse and hold the right button to pan or middle button to move. Hold SHIFT and use the WASD keys to move the camera. Press SHIFT+1 to move faster or SHIFT+2 to move slower. Press SHIFT+R to reset the camera or SHIFT+F to reset the speed.

**Comment by Techjar** [comment]
Thanks, that looks much better. Like I said it still depends on system configuration, and will overflow no matter how hard we try if the user has set a large font size, but we should try to keep things as concise as possible to minimize the effect of that.

File: Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp [hunk 10]
```diff
@@ -273,74 +273,73 @@ void EnhancementsWidget::SaveSettings()
...
+      "Reduces the amount of aliasing caused by rasterizing 3D graphics. This smooths "
+      "out jagged edges on objects.\n\nIncreases GPU load and sometimes causes graphical "
+      "issues. SSAA is more demanding than MSAA, but provides top quality "
+      "geometry anti-aliasing and also applies anti-aliasing to lighting, shader "
```
Comment (Techjar) [inline Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: Put back "significantly", and drop the word "also" as "and also" is not really formal.

File: Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp [hunk 11]
```diff
@@ -190,53 +190,54 @@ void HacksWidget::SaveSettings()
...
-      "games.\n\nIf unsure, leave this checked.");
+      "Ignores any changes to the EFB format.\n\nImproves performance in many games "
+      "without any negative effect. Causes graphical defects in a small "
+      "number of other games.\n\nIf unsure, leave checked.");
   static const char TR_STORE_EFB_TO_TEXTURE_DESCRIPTION[] = QT_TR_NOOP(
-      "Stores EFB Copies exclusively on the GPU, bypassing system memory. Causes graphical defects "
-      "in a small number of games.\n\nEnabled = EFB Copies to Texture\nDisabled = EFB Copies to "
-      "RAM "
-      "(and Texture)\n\nIf unsure, leave this checked.");
+      "Stores EFB Copies exclusively on the GPU, bypassing system memory.\n\nCauses "
```
Comment (Techjar) [inline Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp]: Make this consistent with the XFB copies text.

File: Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp (hunk 9 above)
Comment (8times9) [reply Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp]: That's much better, thanks!

File: Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp (hunk 10 above)
Comment (8times9) [reply Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: I removed "significantly" as that word was all I needed to make your description on Linux not be too long. Hopefully removing the also's will balance out adding back "significantly"? I could remove the "anti-aliasing" from the "applies anti-aliasing to lighting, shader effects..." and it might fit again.

I went ahead and removed any also's from the other description texts, there was only one other.

Comment (Techjar) [reply Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: I checked and it fits fine the way I suggested.

File: Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp [hunk 12]
```diff
@@ -273,74 +273,73 @@ void EnhancementsWidget::SaveSettings()
...
+      "Selects the stereoscopic 3D mode. Stereoscopy allows you to get a better perception "
+      "of depth if you have the necessary hardware. This option heavily decreases emulation "
+      "speed and sometimes causes issues.\n\nSide-by-Side and Top-and-Bottom are "
       "used by most 3D TVs.\nAnaglyph is used for Red-Cyan colored glasses.\nHDMI 3D is "
-      "used when your monitor supports 3D display resolutions.\nHeavily decreases "
-      "emulation speed and sometimes causes issues.\n\nIf unsure, select Off.");
+      "used when your monitor supports 3D display resolutions.\n\nIf unsure, select Off.");
   static const char TR_3D_DEPTH_DESCRIPTION[] =
-      QT_TR_NOOP("Controls the separation distance between the virtual cameras.\nA higher value "
+      QT_TR_NOOP("Controls the separation distance between the virtual cameras.\n\nA higher value "
```
Comment (8times9) [inline Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: Could anyone with more knowledge about this feature tell what a recommended setting would be?

File: Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp [hunk 13]
```diff
@@ -273,74 +273,73 @@ void EnhancementsWidget::SaveSettings()
...
                  "creates a stronger feeling of depth while a lower value is more comfortable.");
   static const char TR_3D_CONVERGENCE_DESCRIPTION[] = QT_TR_NOOP(
       "Controls the distance of the convergence plane. This is the distance at which "
```
Comment (8times9) [inline Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: Ditto

File: Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp (hunk 11 above)
Comment (Techjar) [reply Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp]: I mean in regards to the ordering and line breaks, as well.

File: Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp (hunk 12 above)
Comment (Techjar) [reply Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: It's dependent on screen size, distance from screen, IPD, etc. Basically there is no recommended setting and the user must tweak it to their liking.

Comment (8times9) [reply Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: What about a message along the lines of "Adjust to whichever value provides the best 3D viewing experience."?

Comment (Techjar) [reply Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp]: I don't think any additional information is needed, as the rest of the text explains it pretty concisely.

File: Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp [hunk 14]
```diff
@@ -190,53 +190,54 @@ void HacksWidget::SaveSettings()
...
+      "without any negative effect. Causes graphical defects in a small "
+      "number of other games.\n\nIf unsure, leave checked.");
   static const char TR_STORE_EFB_TO_TEXTURE_DESCRIPTION[] = QT_TR_NOOP(
-      "Stores EFB Copies exclusively on the GPU, bypassing system memory. Causes graphical defects "
-      "in a small number of games.\n\nEnabled = EFB Copies to Texture\nDisabled = EFB Copies to "
-      "RAM "
-      "(and Texture)\n\nIf unsure, leave this checked.");
+      "Stores EFB Copies exclusively on the GPU, bypassing system memory.\nEnabled: EFB "
+      "Copies to Texture\nDisabled: EFB Copies to RAM (and Texture)\n\nCauses "
+      "graphical defects in a small number of games that need to readback from "
```
Comment (MayImilae) [inline Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp]: "that need readback from memory" is rather needless, since the feature isn't explained in detail enough for anyone to know what that means. And we can't really explain everything about the unified memory system and how it differs from PCs here.

Also "readback" is not a very accurate term anyway.

File: Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp [hunk 15]
```diff
@@ -190,53 +190,54 @@ void HacksWidget::SaveSettings()
...
+      "memory.\n\nIf unsure, leave checked.");
   static const char TR_ACCUARCY_DESCRIPTION[] = QT_TR_NOOP(
-      "The \"Safe\" setting eliminates the likelihood of the GPU missing texture updates "
-      "from RAM.\nLower accuracies cause in-game text to appear garbled in certain "
-      "games.\n\nIf unsure, use the rightmost value.");
+      "Adjusts the accuracy at which the GPU receives texture updates from RAM. "
```
Comment (MayImilae) [inline Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp]: Great change! 👍

File: Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp [hunk 16]
```diff
@@ -190,53 +190,54 @@ void HacksWidget::SaveSettings()
...
+      "games.\n\nIf unsure, select the rightmost value.");
 
   static const char TR_STORE_XFB_TO_TEXTURE_DESCRIPTION[] = QT_TR_NOOP(
-      "Stores XFB Copies exclusively on the GPU, bypassing system memory. Causes graphical defects "
-      "in a small number of games that need to readback from memory.\n\nEnabled = XFB Copies to "
-      "Texture\nDisabled = XFB Copies to RAM "
-      "(and Texture)\n\nIf unsure, leave this checked.");
+      "Stores XFB Copies exclusively on the GPU, bypassing system memory. "
+      "\nEnabled: XFB Copies to Texture\nDisabled: XFB Copies to RAM (and Texture)\n\nCauses "
+      "graphical defects in a small number of games that need to readback from memory. "
```
Comment (MayImilae) [inline Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp]: "that need to readback from memory" is not really beneficial, nor accurate.

File: Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp [hunk 17]
```diff
@@ -190,53 +190,54 @@ void HacksWidget::SaveSettings()
...
+      "from RAM.\n\nLower accuracies cause in-game text to appear garbled in certain "
+      "games.\n\nIf unsure, select the rightmost value.");
 
   static const char TR_STORE_XFB_TO_TEXTURE_DESCRIPTION[] = QT_TR_NOOP(
-      "Stores XFB Copies exclusively on the GPU, bypassing system memory. Causes graphical defects "
-      "in a small number of games that need to readback from memory.\n\nEnabled = XFB Copies to "
-      "Texture\nDisabled = XFB Copies to RAM "
-      "(and Texture)\n\nIf unsure, leave this checked.");
+      "Stores XFB Copies exclusively on the GPU, bypassing system memory. "
+      "\nEnabled: XFB Copies to Texture\nDisabled: XFB Copies to RAM (and Texture)\n\nCauses "
```
Comment (MayImilae) [inline Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp]: This restructuring is unwise. The Enabled:/Disabled: subsections should be kept entirely separate from the description itself, as they are callbacks to older settings to assist users in the transition and the intention is to remove them at a later date some point after 6.0.

File: Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp [hunk 18]
```diff
@@ -190,53 +190,54 @@ void HacksWidget::SaveSettings()
...
   static const char TR_IGNORE_FORMAT_CHANGE_DESCRIPTION[] = QT_TR_NOOP(
-      "Ignore any changes to the EFB format.\nImproves performance in many games without "
-      "any negative effect. Causes graphical defects in a small number of other "
-      "games.\n\nIf unsure, leave this checked.");
+      "Ignores any changes to the EFB format.\n\nImproves performance in many games "
+      "without any negative effect. Causes graphical defects in a small "
+      "number of other games.\n\nIf unsure, leave checked.");
   static const char TR_STORE_EFB_TO_TEXTURE_DESCRIPTION[] = QT_TR_NOOP(
-      "Stores EFB Copies exclusively on the GPU, bypassing system memory. Causes graphical defects "
-      "in a small number of games.\n\nEnabled = EFB Copies to Texture\nDisabled = EFB Copies to "
```
Comment (MayImilae) [inline Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp]: This restructuring is unwise. The Enabled:/Disabled: subsections should be kept entirely separate from the description itself, as they are callbacks to older settings to assist users in the transition and the intention is to remove them at a later date some point after 6.0.

File: Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp [hunk 19]
```diff
@@ -190,53 +190,54 @@ void HacksWidget::SaveSettings()
...
-                 "but some games will break.\n\nIf unsure, leave this checked.");
-  static const char TR_VERTEX_ROUNDING_DESCRIPTION[] =
-      QT_TR_NOOP("Rounds 2D vertices to whole pixels. Fixes graphical problems in some games at "
-                 "higher internal resolutions. This setting has no effect when native internal "
-                 "resolution is used.\n\nIf unsure, leave this unchecked.");
+      "Uses a less accurate algorithm to calculate depth values.\n\nCauses issues in a few "
+      "games, but can give a decent speed increase depending on the game and/or your "
+      "GPU.\n\nIf unsure, leave checked.");
+  static const char TR_DISABLE_BOUNDINGBOX_DESCRIPTION[] = QT_TR_NOOP(
+      "Disables the bounding box emulation.\n\nThis may improve the GPU performance a lot, "
```
Comment (MayImilae) [inline Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp]: The "the" is needless here. Suggesting:

>Disabled bounding box emulation

File: Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp [hunk 20]
```diff
@@ -142,62 +142,60 @@ void AdvancedWidget::OnEmulationStateChanged(bool running)
...
-      QT_TR_NOOP("Render the scene as a wireframe.\n\nIf unsure, leave this unchecked.");
+      QT_TR_NOOP("Renders the scene as a wireframe.\n\nIf unsure, leave unchecked.");
   static const char TR_SHOW_STATS_DESCRIPTION[] =
-      QT_TR_NOOP("Show various rendering statistics.\n\nIf unsure, leave this unchecked.");
-  static const char TR_TEXTURE_FORMAT_DECRIPTION[] =
-      QT_TR_NOOP("Modify textures to show the format they're encoded in. Needs an emulation reset "
-                 "in most cases.\n\nIf unsure, leave this unchecked.");
+      QT_TR_NOOP("Shows various rendering statistics.\n\nIf unsure, leave unchecked.");
+  static const char TR_TEXTURE_FORMAT_DECRIPTION[] = QT_TR_NOOP(
+      "Modifies textures to show the format that they are encoded in.\n\nMay require "
```
[the remark this answers is not in the document]
Comment (legoj15) [reply Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp]: From my experience, an emulation resent isn't required, so that may be why it was changed
